call-home/src/common/constants.rs

```diff
@@ -1 +1,15 @@
+/// PRODUCT is the name of the project for which this call-home component is deployed.
 pub(crate) const PRODUCT: &str = "Mayastor";
+
+/// DATA_DIR_PATH is the directory path for the temporary files generated during
+/// encryption.
+pub(crate) const DATA_DIR_PATH: &str = "/encryption_dir";
+
+/// ENCRYPTION_KEY_FILEPATH is the path to the encryption key.
+pub(crate) const ENCRYPTION_KEY_FILEPATH: &str = "/key/";
+
+/// RECEIVER_API_ENDPOINT is the URL to anonymous call-home metrics collection endpoint.
+pub(crate) const RECEIVER_API_ENDPOINT: &str = "";
```

Comment:
This is blank until we have the endpoint and then we'll add it right?

Reply:
Yes, that is correct.